server/game/cards/03-WC/ChantOfHubris.js

```diff
@@ -0,0 +1,29 @@
+const Card = require('../../Card.js');
+
+class ChantOfHubris extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            condition: context => context.game.creaturesInPlay.length > 1 && context.game.creaturesInPlay.some(card => card.hasToken('amber')),
```

Comment:
@jeremylarner can correct me if I'm wrong but I think this is redundant as it's duplicated in the target: below

Reply:
@cryogen: Not really. Since there is an action so select a creature with amber and an action to select another creature, we need a broader condition to validate the whole card has condition to be fulfilled.